src/browser/modules/D3Visualization/GraphEventHandler.ts

```diff
@@ -55,7 +55,7 @@ export class GraphEventHandler {
     }
     this.selectedItem = item
     item.selected = true
-    this.graphView.update()
+    this.graphView.update({ updateNodes: true, updateRelationships: true })
```

Comment:
What do you think of having both these options as true by default, so you can just call .update() when you want a full update, and then optionally use the parameter when needed?